src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/index.js

```diff
@@ -0,0 +1,6 @@
+import DownloadButton from './DownloadButton';
+import ToolbarButton from './ToolbarButton';
+
+export {
+  DownloadButton, ToolbarButton,
+};
```

Comment:
```suggestion
export { default as DownloadButton } from './DownloadButton';
export { default as ToolbarButton } from './ToolbarButton';
```

Reply:
does this change how I import these modules?

Reply:
Not in the slightest. This syntax is just a more concise version of what you wrote, without bridging the referred file without bringing their code into this context.